CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ See [RELEASE](./RELEASE.md) for workflow instructions.
 
 ### Upgrade information
 
+This upgrade is incompatible with versions older than v1.1.6.
+
+At the start of the upgrade, mesh data will be pruned and compacted/vacuumed. Pruning takes longer for
+nodes that joined the network earlier. For 1-week-old nodes, it takes ~20 minutes. for 3-week-old
+nodes it takes ~40 minutes. Vacuum takes ~5 minutes. The vacuum operation requires extra disk space
```

Comment:
```suggestion
nodes that joined the network earlier. For 1-week-old nodes, it takes ~20 minutes. for 3-week-old
nodes it takes ~40 minutes. The vacuum operation requires extra disk space
```

That sentence seems out of place

Reply:
how so? i was trying to state that vacuum operation takes 5 minutes.

Reply:
changed a little. hopefully its less awkward

Reply:
I was confused by the stated durations.

I assume pruning takes up to 40 minutes for 3 week old nodes and afterwards it takes another 5 minutes to vacuum independently of how old the node is? I wrongly read it as "takes 40 minutes to vacuum - vacuum takes 5 minutes". The changed text is clearer 🙂